taproot-updates.md

```diff
@@ -161,15 +160,15 @@ The commitment transaction will then become:
             # NB: we also need the remote signature, which prevents us from unilaterally changing the HTLC-success transaction
             OP_HASH160 <RIPEMD160(payment_hash)> OP_EQUALVERIFY
             <remote_htlcpubkey> OP_CHECKSIGVERIFY <local_htlcpubkey> OP_CHECKSIG
-            1 OP_CHECKSEQUENCEVERIFY
           ",
           "tapleaf_2": "
             # funds go to the remote node after an absolute delay (timeout)
             <remote_htlcpubkey>
-            OP_CHECKSIGVERIFY
+            OP_CHECKSIG
+            OP_CHECKSEQUENCEVERIFY
             <cltv_expiry>
             OP_CHECKLOCKTIMEVERIFY
-            1 OP_CHECKSEQUENCEVERIFY
+            OP_DROP
```

Comment:
this script would end with 2 top stack elements

this works, there are also other ways with scripts the same size but this one looks like the one earlier mentioned with a separate `<cltv_expiry> OP_CHECKLOCKTIMEVERIFY OP_DROP`